styles.css

```diff
@@ -736,11 +826,588 @@ body {
   .dsk-menu {
     display: none;
   }
+
+  #r-tag {
+    display: none;
+  }
+
+  .cursor {
+    display: none;
+  }
 }
 
 @media (min-width: 768px) {
+  .body {
+    width: 1440px;
+  }
+
+  .app-bar {
+    width: 1440px;
+    height: 72px;
+    left: 0;
+    top: 0;
+  }
+
+  .main {
+    width: 100%;
+    height: 916px;
+    left: 0;
+    top: 1px;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: auto;
+    justify-content: center;
+    margin: 0;
+  }
+
+  .header-2 {
+    border-radius: 0;
+    height: 916px;
+    background-image: url(img/Header\ bg.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 0 0 142px 0;
+  }
```

Comment:
- [ ] Please make sure that the text and social media icon links in the headline are in the center of the page, to follow the figma design and show your ability to parse a Figma design to create a UI.

- [ ] Kindly add some margin between the text and social media icons to follow the figma design as requested in the project requirement.

### Your design
![Screenshot from 2023-04-07 04-16-25](https://user-images.githubusercontent.com/91142432/230541863-4cf53dc6-236f-4fb3-9657-05a14be5d7f7.png)

### What it's supposed to look like
![Screenshot from 2023-03-31 15-15-20](https://user-images.githubusercontent.com/91142432/230541959-312bc2b6-021d-446b-9eda-91807fa2166f.png)